internal/kafka/internal/services/util.go

```diff
@@ -70,8 +69,8 @@ func BuildKeycloakClientNameIdentifier(kafkaRequestID string) string {
 	return fmt.Sprintf("%s-%s", "kafka", strings.ToLower(kafkaRequestID))
 }
 
-func BuildCustomClaimCheck(kafkaRequest *dbapi.KafkaRequest, ssoconfigProvider keycloak.SSOProvider) string {
-	if ssoconfigProvider == keycloak.REDHAT_SSO {
+func BuildCustomClaimCheck(kafkaRequest *dbapi.KafkaRequest, ssoconfigProvider string) string {
+	if ssoconfigProvider == "redhat_sso" {
```

Comment:
Why this change?

Reply:
Hey @machi1990 this change is in relation to question 3 here. https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1007#issuecomment-1119402717